research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -0,0 +1,62 @@
+
+Breadcrumbs are navigation land marks used to show a user's location within the site or app.
+
+## Use Cases
+
+A `<breadcrumb />` trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page's main content.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/breadcrumb/)
+- [Auth0](https://styleguide.auth0.com/components/breadcrumb)
+- [Atlas Kit](https://atlassian.design/components/breadcrumbs/examples)
+- [Carbon](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+- [Lightning](https://www.lightningdesignsystem.com/components/breadcrumbs/)
+- [Semantic UI](https://semantic-ui.com/collections/breadcrumb.html)
+
+## &lt;breadcrumb&gt; Properties <a href="#breadcrumb-properties" id="breadcrumb-properties"></a>
+
+| Attribute Name | Type             | Default Value | Description                                                                      |     |
+| -------------- | ---------------- | ------------- | -------------------------------------------------------------------------------- | --- |
+| `items`        | `HTMLCollection` |               | A `HTMLCollection` of items to represent the current path within the application |     |
+| `divider`      | `custom`         | `/`           | Represnets the custom item to be used as divider between the items               |     |
+
+## &lt;breadcrumb-divider&gt; Properties <a href="#breadcrumb-divider-properties" id="breadcrumb-divider-properties"></a>
+
+| Property Name | Type     | Default Value | Description                                          |
+| ------------- | -------- | ------------- | ---------------------------------------------------- |
+| `content`     | `custom` | `/`           | The content to be rendered as boundary for the items |
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+#### Diagram <a class="link" href="#diagram" id="diagram"></a>
+
+<BreadcrumbAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<breadcrumb>` - The root element that contains the list of items and dividers **[required]**
+- `<breadcrumb-item>` - The element representing a place in the path to the current position on the application **[required]**
+- `<breadcrumb-divider>` - The divider between the `<breadcrumb-item />`
+
+#### Accessibility <a class="link" href="#accessibility" id="accessibility"></a>
+
+It should be wrapped by a navigation landmark `nav` containing `aria-label` or `aria-labelledby`, the `<breadcrumb-item />` representing the current page should contain `aria-current="page"` ( if the current page isn't a link the attribute is optional )
```

Comment:
Can we get very specific about the parts. eg:

part=breadcrumb:
* role = nav

I'm also curious on the name computation here. Is it common for the breadcrumb to be what you focus on or solely the first link? If you land on the container and it is focusable, how does it compute the name?

Reply:
The commons is for the container not be focusable and you always land in the first link itself

Reply:
> It should be wrapped

Can you define what `it` is

Also, can you add the part attribute to your items so that your HTML maps to your anatomy?

Reply:
Done